wave-app/src/plot.py

```diff
@@ -1,25 +1,92 @@
-from h2o_wave import main, app, Q, ui
+import numpy as np
 import pandas as pd
 import plotly
 import plotly.graph_objects as go
 from plotly.missing_ipywidgets import FigureWidget
 
-# Use plotly to make scatter plot for dataset description.
-def make_scatter_plot(q:Q, df: pd.DataFrame):
-    fig = go.Figure(
-        data=go.Scattergeo(lat=df['latitude'], lon=df['longitude']),
-        layout=go.Layout(
-            title='Australia Fire Daily', title_font_color = '#FEC925',
-            paper_bgcolor='rgba(0, 0, 0, 0)',
-            margin_r=10, margin_l=10, margin_t = 50, margin_b=10
-    )).update_geos(
-        fitbounds='locations'
+AUS_LAT_RANGE = (-40, -9)
+AUS_LON_RANGE = (112, 155)
+
+GEO = dict(
+    showland=True,
+    landcolor="rgb(128, 128, 128)",
+    subunitcolor="rgb(255, 255, 255)",
+    countrycolor="rgb(255, 255, 255)",
+    showlakes=True,
+    lakecolor="rgb(0, 0, 0)",
+    showrivers=True,
+    rivercolor="rgb(0, 0, 0)",
+    showocean=True,
+    oceancolor="rgb(0, 0, 0)",
+    showsubunits=True,
+    showcountries=True,
+    resolution=50,
+    lonaxis=dict(
+        showgrid=True,
+        gridwidth=0.5,
+        range=AUS_LON_RANGE,
+        dtick=5
+    ),
+    lataxis=dict(
+        showgrid=True,
+        gridwidth=0.5,
+        range=AUS_LAT_RANGE,
+        dtick=5
     )
-    fig.update_traces(marker_color='#E25822')
+)
+
 
+# Use plotly to make scatter plot for dataset description.
+def show_bush_fires(df: pd.DataFrame):
+    aus_fires = df.copy()
+    aus_fires.latitude = aus_fires.latitude.round(1)
+    aus_fires.longitude = aus_fires.longitude.round(1)
+    sample = aus_fires[(aus_fires.acq_date >= '2019-09-01') & (aus_fires.acq_date <= '2020-03-31')]
+    sample = sample.groupby(['latitude', 'longitude']).size().reset_index()
+    sample.columns = ['latitude', 'longitude', 'fire_cnt']
+    sample = sample[sample.fire_cnt > 3]
+    fig = go.Figure(data=go.Scattergeo(
+        lat=sample['latitude'],
+        lon=sample['longitude'],
+        marker=dict(
+            color=sample['fire_cnt'],
+            reversescale=True,
+            opacity=0.8,
+            line=dict(width=0),
+            size=np.log(sample.fire_cnt) + 1,
+        ),
+    ))
+    _ = fig.update_layout(
```

Comment:
it may be ok to not capture the return value if it's not used (`_`). You could just do `fig.update(...)`